### PR TITLE
Specify target path caching for resources.ExecuteAsTemplate

### PR DESCRIPTION
**Pull request by willfaught**

### CODE REVIEW

File: content/en/hugo-pipes/resource-from-template.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ signature: ["resources.ExecuteAsTemplate TARGET_PATH CONTEXT RESOURCE"]
 
 In order to use Hugo Pipes function on an asset file containing Go Template magic the function `resources.ExecuteAsTemplate` must be used.
 
-The function takes three arguments: the target path for the created resource, the template context, and the resource object.
+The function takes three arguments: the target path for the created resource, the template context, and the resource object. The target path is used to cache the result.
```
Comment (jmooring) [inline content/en/hugo-pipes/resource-from-template.md:22]:
```suggestion
The result is cached using the target path as the cache key.
```

**Review by jmooring — CHANGES_REQUESTED** [review]

File: content/en/hugo-pipes/resource-from-template.md (hunk 1 above)
Comment (willfaught) [reply content/en/hugo-pipes/resource-from-template.md:22]: I found it useful for it to explain what the first param is. It would be too bad to lose that.

Comment (jmooring) [reply content/en/hugo-pipes/resource-from-template.md:22]: Sorry, I meant to replace just the last sentence.

Comment (willfaught) [inline content/en/hugo-pipes/resource-from-template.md:22]:
```suggestion
The function takes three arguments: the target path for the created resource, the template context, and the resource object. The result is cached using the target path as the cache key.
```

Comment (willfaught) [reply content/en/hugo-pipes/resource-from-template.md:22]: @jmooring How's that?

Comment (jmooring) [reply content/en/hugo-pipes/resource-from-template.md:22]: Thank you.

**Comment by jmooring** [comment]
@willfaught If you have a moment, resources.FromString is also cached, using the target path as the cache key. Make similar clarification there?

**Comment by willfaught** [comment]
Sure thing, will do it today or tomorrow.

**Pull request merged**